### PR TITLE
feat: add Acknowledgements component to render in Rule page

### PR DESCRIPTION
**Pull request by offsetChris**
Addresses Issue #35 

Not completely sure if this is how you want to handle this issue. Was trying to come up with a non-breaking change that can accommodate what is out there currently and reuse the existing function. Open to suggestions. Also unsure about how to tell graphQL I added a new field to pass the build here.

If this works will need to add corresponding previous_authors to rules repo rule templates.

### CODE REVIEW

**Comment by jkodu** [comment]
@offsetChris, thanks for this. I have put together a PR in the rules repo for this change - https://github.com/act-rules/act-rules.github.io/pull/997.

File: src/components/acknowledgements.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+ *
+ * @param {Object} props Props
+ */
+const Acknowledgements = ({ scrollLinkId, items, contributors }) => {
+	return (
+		<>
+			<a id={scrollLinkId} href={`#${scrollLinkId}`}>
+				<h2>Acknowledgements</h2>
+			</a>
+			{Object.keys(items).map(key => {
```
[the remark this answers is not in the document]
Comment (WilcoFiers) [reply src/components/acknowledgements.js]: Nope. JS doesn't give any guarantees for the order of keys in an object. Object.keys() can return the array in any order that's most suitable to the JS runtime. Even if you put them in order in the frontmatter, there's no guarantee it'll come out the same way in `Object.keys()`. It just does that most of the time.

File: src/components/acknowledgements.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+				const values = items[key] || []
+				const heading = key.split('_').join(' ')
```
[the remark this answers is not in the document]
Comment (WilcoFiers) [reply src/components/acknowledgements.js]: The CSS won't be there when we take this to the W3C. Also, it's bad practice. Don't rely on CSS to fix problems in your HTML. Get the HTML right, then apply CSS.

**Comment by jkodu** [comment]
@offsetChris can you handle the above 2 change requests from @WilcoFiers?

**Pull request merged**